api/handlers/slack.ts

```diff
@@ -24,14 +24,19 @@ export async function buildEntitledBranchList(entitlement: any, repoBranchesRepo
     const [repoOwner, repoName, directoryPath] = repo.split('/');
     const branches = await repoBranchesRepository.getRepoBranches(repoName, directoryPath);
     for (const branch of branches) {
-      let buildWithSnooty = true;
       if ('buildsWithSnooty' in branch) {
```

Comment:
I know this is a holdover from the past, but I think we can get rid of this conditional. It doesn't gate anything and we're just checking true/false which undefined will work for as well.